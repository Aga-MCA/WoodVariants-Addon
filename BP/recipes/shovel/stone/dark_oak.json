{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "stone_dark_oak_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "minecraft:cobblestone"
      }
    },
    "result": {
      "item": "aga_wood_variants:stone_dark_oak_shovel"
    }
  }
}