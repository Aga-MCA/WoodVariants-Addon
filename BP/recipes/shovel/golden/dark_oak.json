{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_dark_oak_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_dark_oak_shovel"
    }
  }
}